src/xdocs-examples/resources/com/puppycrawl/tools/checkstyle/checks/avoidescapedunicodecharacters/Example5.java

```diff
@@ -0,0 +1,31 @@
+/*xml
+<module name="Checker">
+  <module name="TreeWalker">
+    <module name="AvoidEscapedUnicodeCharacters">
+      <property name="allowNonPrintableEscapes" value="true"/>
+    </module>
+  </module>
+</module>
+*/
+
+package com.puppycrawl.tools.checkstyle.checks.avoidescapedunicodecharacters;
+
+// xdoc section -- start
+public class Example5 {
+  // OK, a normal String below.
+  String unitAbbrev = "μs";
+  // violation below, printable escape character. 'should be avoided.'
+  String unitAbbrev1 = "\u03bcs";
+  // violation below, printable escape character. 'should be avoided.'
+  String unitAbbrev2 = "\u03bc\u03bc\u03bc";
+  // violation below
+  String unitAbbrev3 = "\u03bcs"; // it is μs
+  // violation below
+  String unitAbbrev4 = "\u03bc\u03bcs";
+  public static int content() {
+    char content = 'r';
+    // OK, non-printable escape character below
+    return '\ufeff' + content;
+  }
+}
+// xdoc section -- end
```

Comment:
Please copy this example file code to all others.
And change only comments, do not remove comments, amount of lines must be the same in all Example classes.

In other words `tring unitAbbrev2 = "\u03bc\u03bc\u03bc` must be in each file 

In other words if you compare two examples side by side, other differences should be in : config, and comment above code, and name of top level class.